tusker/config.py

```diff
@@ -46,6 +47,13 @@ def get(self, name, type, required=False, default=None):
             else:
                 return default
         value = self.data[name]
+        if isinstance(value, str) and re.match(r"^\${[a-zA-Z_]+}$", value):
```

Comment:
It would be nice if the code also supported multiple interpolations per configuration string. You could use [re.sub](https://docs.python.org/3/library/re.html#re.sub) for that with a replacement function.

Reply:
Yep, that's nicer. Updated it with an re.sub function.